__tests__/getCategories.test.js

```diff
@@ -37,4 +37,40 @@ describe("app ", () => {
         });
     });
   });
+  describe("/api/reviews/:review_id", () => {
+    test("respond with a status of 200 and display a review object that match with ID endpoint", () => {
+      return request(app)
+        .get("/api/reviews/1")
+        .expect(200)
+        .then((review) => {
```

Comment:
I think the ticket for this endpoint asks for an object rather than an array, so I'd refactor this to fulfil the requirements of the ticket